src/mondeath.cpp

```diff
@@ -405,7 +405,17 @@ void mdeath::worm( monster &z )
 
 void mdeath::disappear( monster &z )
 {
-    add_msg_if_player_sees( z, m_good, _( "The %s disappears." ), z.name() );
+    // The .compare() method returns 0 if equal
+    // Any other value is implicitly true
+    // so we need to reverse it
+    // substring(x,y) returns the y-character string starting at index x
+    bool begins_with_your = !( z.name().substring(0,4).compare("your") );
+    // this bool is now true if the enemy's name begins with 'your'
+
+    std::string msg = ( begins_with_your ) ?
+        "%s disappears." : "The %s disappears.";
```

Comment:
These are no longer translated.

Reply:
Hmm. Enclosing the `msg` in the translation marker won't do it?

Reply:
I dropped the translation markers around the two strings, and then passed the string directly to `add_msg_if_player_sees`.

Reply:
`_()` does two jobs.
1. It returns the translated version of the string passed to it, if such a version exists.
2. It marks strings for extraction, so the translators can make translated versions of it.